src/reconcile/rec_visibility.c

```diff
@@ -344,8 +344,6 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, v
                 upd_select->stop_ts = upd->start_ts;
             if (upd->txnid != WT_TXN_NONE) {
                 upd_select->stop_txn = upd->txnid;
-                if (upd->start_ts == WT_TS_NONE)
```

Comment:
Why we don't need to set upd_select->stop_ts to WT_TS_NONE here? I think it is set to WT_TS_MAX by default.

Reply:
Fixed.